tests/e2e/run-e2e-tests.sh

```
@@ -0,0 +1,458 @@
+    mkdir -p "$TEST_RESULTS_DIR"
+
+    # Install test polkit action definitions
+    if [ -f "$SCRIPT_DIR/org.quickshell.polkit.test.policy" ]; then
+        sudo cp "$SCRIPT_DIR/org.quickshell.polkit.test.policy" /usr/share/polkit-1/actions/
+        log_info "Installed test polkit actions"
+    fi
+
+    # Start D-Bus session bus if not running
+    if [ -z "$DBUS_SESSION_BUS_ADDRESS" ]; then
+        log_info "Starting D-Bus session bus..."
+        eval $(dbus-launch --sh-syntax)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)